### ISSUE TITLE
The n_val violation error message is confusing when R > N

### ISSUE THREAD

**Issue by russelldb** [basho/riak_kv#398]
See https://github.com/basho/riak_kv/blob/master/src/riak_kv_wm_object.erl#L1023

In this case the user is always told that there w values are wrong, even on a fetch. Try

```
curl localhost:8098/riak/test/foo?r=10
```

on a new cluster for response of

```
Specified w/dw/pw values invalid for bucket n value of 3
```

Thanks Ingo, for report http://riak-users.197444.n3.nabble.com/Data-loads-td4025122.html#a4025385

**Comment by reiddraper** [comment]
Is this a 2.0 blocker? @seancribbs, @russelldb?

**Comment by seancribbs** [comment]
@reiddraper No, but I'll be hitting some minor client-related nuisances and bugs this week. This is a small thing to fix.

**Comment by reiddraper** [comment]
> This is a small thing to fix.

No doubt, but we have a ton of 'small things to fix' for 2.0. Gotta draw the line somewhere. It's fine if this is inside the 2.0 line, but I want to think critically about what makes it in.

**Comment by seancribbs** [comment]
I'm not incredibly married to either decision, I was only basing this on my capacity. Let's bump to 2.0.1 or 2.1.

**Comment by reiddraper** [comment]
Thanks @seancribbs. Moved to 2.0.1.
